### ISSUE TITLE
Retrain BEVDet Using TIER4 Dataset

### ISSUE THREAD

**Issue by cyn-liu** [autowarefoundation/autoware.universe#8036]
### Checklist

- [x] I've read the [contribution guidelines](https://github.com/autowarefoundation/autoware/blob/main/CONTRIBUTING.md).
- [x] I've searched other issues and no duplicate issues were found.
- [x] I've agreed with the maintainers that I can plan this task.

### Description

Due to the poor detection performance of the current BEVDet model on the TIER4 dataset, we plan to retrain the BEVDet model on the TIER4 dataset.

Reference this BEVDet train code baseline: [Link](https://github.com/LCH1238/BEVDet)
TIER4 Dataset  [Link](https://drive.google.com/file/d/1UaarK88HZu09sf7Ix-bEVl9zGNGFwTVL/view?usp=sharing)

Note: Now, the data provided by the above link only has 39 scenes, totaling 2874 keyframes, which does not meet the training requirements. We will first use the TIER4 dataset to complete the training process.

### Purpose

Retrain the BEVDet model on the TIER4 dataset.

### Possible approaches

1. Use the same network configuration as the deployed onnx model to train on the Nuscenes dataset to ensure that the model can converge normally.
2. train on the TIER4 dataset.

### Definition of done

to write a script for converting TIER4 dataset to BEVDet train pkl file format

**Comment by cyn-liu** [comment]
We retraining BEVDet with TIER4 dataset.
The statistical results  of TIER4 dataset are as follows:
39 scenes(approximately 2600 keyframes), divide into 31 train sets and 7 val sets, and 1 scene discarded(due to missing files) .

After `20 epochs` of training, the loss converges. The model(tensorrt model) is able to detect some objects in a single frame of TIER4 data.
a frame from train set:
![bevdet_20](https://github.com/user-attachments/assets/dd4e4465-65fe-40b7-9947-103bb346e24d)

a frame from val set:
![bevdet_20_val1](https://github.com/user-attachments/assets/40ad5177-3b92-41ca-9731-f2e7e68feafb)

but  the evaluation results of the model on val sets are very poor.
```
mAP: 0.0066
mATE: 0.9937
mASE: 0.7847
mAOE: 0.8268
mAVE: 0.8569
mAAE: 0.7370
NDS: 0.0834
Eval time: 27.3s

Per-class results:
Object Class	AP	ATE	ASE	AOE	AVE	AAE
car	0.022	0.599	0.193	0.240	1.111	0.658
truck	0.014	0.939	0.332	0.100	0.665	0.237
bus	0.000	1.000	1.000	1.000	1.000	1.000
trailer	0.000	1.000	1.000	1.000	1.000	1.000
construction_vehicle	0.000	1.000	1.000	1.000	1.000	1.000
pedestrian	0.000	1.000	1.000	1.000	1.000	1.000
motorcycle	0.030	1.399	0.322	1.101	0.079	0.000
bicycle	0.000	1.000	1.000	1.000	1.000	1.000
traffic_cone	0.000	1.000	1.000	nan	nan	nan
barrier	0.000	1.000	1.000	1.000	nan	nan
```
The possible reason is that the amount of train dataset is too small, and the model has poor generalization on the validation set.

**Comment by cyn-liu** [comment]
Organizing the model retraining process in this [repos](https://github.com/cyn-liu/BEVDet/tree/train_export)

**Comment by cyn-liu** [comment]
The detection results of TIER4 data on a model trained on the NuScenes dataset:
![image](https://github.com/user-attachments/assets/1ea2deb9-31bc-4afd-ad25-9272e7454cd9)
